nuxeo-drive-client/nxdrive/data/ui5/js/ndrive-conflicts.js

```diff
@@ -54,16 +54,27 @@ ConflictsController.prototype.updateErrors = function($scope) {
 	$scope.ignoreds = ignoreds;
 }
 ConflictsController.prototype.updateConflicts = function($scope, $interval) {
+    var lastContributorPending = false;
 	self = this;
 	$scope.conflicts = angular.fromJson(drive.get_conflicts());
 	for (var i=0; i<$scope.conflicts.length; i++) {
+		lastContributorPending = lastContributorPending || $scope.conflicts[i].need_refresh;
 		if ($scope.conflicts[i].last_error == "DUPLICATING") {
 			setTimeout( function() {
 				self.updateConflicts($scope, $interval);
 				$scope.$apply();
 			}, 1000);
 		}
 	}
+	if (lastContributorPending) {
```

Comment:
Here is a sample response for GetFileSystemItem (from cpodev1):
`{
  "digestAlgorithm": "MD5",
  "canUpdate": true,
  "downloadURL": "nxbigfile/default/b18d7c7c-221f-4783-b9d4-794fa71bcce4/blobholder:0/fiddler4setup.exe",
  "digest": "07ada6a98fb9eae1db2284be2034d9c9",
  "creationDate": 1467103711783,
  "folder": false,
  "creator": "336",
  "lastContributor": "336",
  "lastModificationDate": 1467103711783,
  "parentId": "userSyncRootParentFactory#default#b318ba74-d734-4825-b506-130471d168de",
  "canRename": true,
  "canDelete": true,
  "lockInfo": null,
  "name": "fiddler4setup.exe",
  "id": "defaultFileSystemItemFactory#default#b18d7c7c-221f-4783-b9d4-794fa71bcce4",
  "path": "/org.nuxeo.drive.hierarchy.permission.factory.PermissionTopLevelFactory#/userSyncRootParentFactory#default#b318ba74-d734-4825-b506-130471d168de/defaultFileSystemItemFactory#default#b18d7c7c-221f-4783-b9d4-794fa71bcce4",
  "userName": "336"
}`
The lastContributor value is 336 not "Raja D G". So after calling GetFileSystemItem API, DS client needs to resolve '336' to 'Raja D G' using URL "https://cpodev1-dm.sharpb2bcloud.com/app1/api/v1/user/336".
If we resolve the userid to username while syncing the file, then we are making an extra call while syncing to resolve user name, which results in slowing down of the sync process.